client/packages/invoices/src/Prescriptions/DetailView/PrescriptionLineEdit/PrescriptionLineEditForm.tsx

```diff
@@ -229,11 +230,12 @@ export const PrescriptionLineEditForm: React.FC<
           />
           <Grid container>
             <ModalLabel label={t('label.issue')} />
-            <NonNegativeIntegerInput
+            <NumericTextInput
               autoFocus
               value={issueQuantity}
               onChange={handleIssueQuantityChange}
-              defaultValue={0}
+              defaultValue={1}
```

Comment:
the defaultValue doesn't show initially - when I add an item the Issue quantity is `0`
Which is good - it actually shouldn't default to `1` at that point - if it defaults to a valid number then we don't know if the user has forgotten to enter an issue quantity and cannot alert. we'd save the prescription line with a quantity of 1 which might not be correct

Reply:
Yeah, the `defaultValue` will only show up if the `value` prop is not yet defined. And in this case, it's being set to `0` in the initial `useState`, so no need for `defaultValue` here at all. Have removed :)